modules/portmaster/src/i18n/helptexts.yaml

```diff
@@ -41,95 +40,92 @@ uiMode:
-    default settings for all apps.
+    Here you can set system-wide preferences and configure default rules for all your apps and connections.
 
-    Any settings you have changed in any app settings will override
-    the Global settings that has been changed by you.
+    It is easy to create a stricter global ruleset and then create exceptions in the app settings, which override the global default.
 
 #########################
 ### Per-App Settings Page
 
 appSettings:
   title: App Settings
   content: |
-    By default the settings that are set in the global settings
-    will be used by the app by default. If you change the app
-    settings, it will override the global settings for that
-    specific app.
-  nextKey: appSettings-Filter
+    Here you can configure app-specific settings which override the global settings.
+
+    It is easy to create a stricter global ruleset and then create exceptions in the app settings, which override the global default.
 
 appSettings-Filter:
   title: Display Mode
   content: |
-    This allows you to easily change the view of settings.
+    Quickly change what settings are displayed:
 
-    View Active (Default):<br>
-    This shows only the settings that you have activated.
+    **View Active:**<br>
+    Only show app-specific settings which override the global default.
 
-    View All:<br>
-    This shows all the settings that you activated and not
-    activated.
+    **View All:**<br>
+    Show all settings. App-specific settings which override the global default are highlighted.
 
 #########################
 ### Configuration Options
 
 networkRatingSwitch:
   title: Network Rating Switch
   content: |
-    These switches allow you to enable the selected setting for the
-    network rating levels. Ie, Trusted, Untrusted and Danger.
-
-    - **Enabled** *(Green)*  <br />
-      The setting is enabled for that network rating level.
-
-    - **Disabled** *(Gray)* <br />
-      The setting is disabled for that network rating level.
-
-    - **Off** *(Red)* <br />
-      The setting is off for all network rating levels. <br />
-      (Note: Not all network level settings can be fully turned off in
-      order to provide a protective baseline for the danger level)
-    Off(Red)
-    The setting is off for all network rating levels.
-    (Note: Not all network level settings can be fully turned off in
-    order to provide a protective baseline for the danger level)
+    Choose in which environment(s) this setting should be enforced.
+
+    Easily configure settings for different environments. When changing the "Network Rating" in the top left all settings will immediately adjust to your preferences.
+
+    - **Off**: This setting is completely turned off and not enforced anywhere
+    - **Danger**: This setting is only enforced when you set your Network Rating to "Danger"
+    - **Untrusted**: This setting is both enforced in the "Danger" _AND_ in the "Untrusted" environment
```

Comment:
I feel "enforced" is too hard. It's just enabled. A global setting will not be enforced on a per-app level.